web/toolbar.js

```diff
@@ -70,6 +73,16 @@ class Toolbar {
       },
       { element: options.download, eventName: "download" },
       { element: options.viewBookmark, eventName: null },
+      {
+        element: options.editorFreeTextButton,
+        eventName: "switchannotationeditormode",
+        eventDetails: { mode: AnnotationEditorType.FREE_TEXT },
+      },
+      {
+        element: options.editorSelectButton,
+        eventName: "switchannotationeditormode",
+        eventDetails: { toggle: true },
+      },
```

Comment:
Please move this block up, and change this to:
```suggestion
      {
        element: options.editorNoneButton,
        eventName: "switchannotationeditormode",
        eventDetails: { mode: AnnotationEditorType.NONE },
      },
```